Gemfile

```diff
@@ -51,6 +51,7 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rails-controller-testing'
   gem 'rspec-rails'
 end
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214216889-44b9f2ca-dd2a-4d3a-b3aa-a591668864d2.png)

- [ ] Your link to your license file is not working its because you didn't create a license file in your repository by following these guidelines to create a license file given by microverse :thinking:

**Note: You can use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to add a license file using Github License Feature :+1:**